db_schema/migrations/10_anchor_episodes_page.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS anchorEpisodesPage (
+  account_id INTEGER NOT NULL,
+  episode_id VARCHAR(128) NOT NULL,
+  web_episode_id VARCHAR(128) NOT NULL,
+  PRIMARY KEY (account_id, episode_id, web_episode_id)
```

Comment:
Shall we add more fields here? I thought about the date, but technically, the IDs should be unique and the date of insertion wouldn't add much value. Couldn't come up with anything else that might be interesting.